src/views/sources/viewSourcesList.tsx

```diff
@@ -372,7 +387,10 @@ const SourcesListView: React.FunctionComponent = () => {
                   <ActionMenu<SourceType>
                     item={source}
                     actions={[
-                      { label: t('table.label', { context: 'edit' }), onClick: onEditSource },
+                      {
+                        label: t('table.label', { context: 'edit' }),
+                        onClick: onEditSource
+                      },
```

Comment:
Just for the record, it looks like [the very next line in the file](https://github.com/quipucords/quipucords-ui/blob/main/src/views/sources/viewSourcesList.tsx#L376) is already broken into one-key-per-line format, but it doesn't need to - I can move the object back into single line (105 chars long), and `test:lint` will not complain. `test:lint-fix` is leaving both as they are.

I can modify that other line as part of that PR since I already modify the file, so both are consistent. Or I can leave everything as is.